sources/views/contacts.js

```diff
@@ -62,4 +72,21 @@ export default class Contacts extends JetView {
 		this.list.unselectAll();
 		this.show("/top/contacts/contactsForm");
 	}
+
+	contactsFilter() {
+		const value = this.$$("contactsFilterInput").getValue().toLowerCase();
+		this.list.filter((contact) => {
+			if (
+				contact.value.toLowerCase().indexOf(value) !== -1 ||
+				contact.Email.toLowerCase().indexOf(value) !== -1 ||
+				contact.Skype.toLowerCase().indexOf(value) !== -1 ||
+				contact.Job.toLowerCase().indexOf(value) !== -1 ||
+				contact.Company.toLowerCase().indexOf(value) !== -1 ||
+				webix.i18n.longDateFormatStr(contact.Birthday).toLowerCase().indexOf(value) !== -1 ||
+				contact.Address.toLowerCase().indexOf(value) !== -1) {
+				return contact;
```

Comment:
If only these keys exists in contact you can use shorter version

return Object.keys(contact).some((key) => contact[key].indexOf(value) !== -1);

Reply:
also add one if statement for birthday

Reply:
Contact have more options, like Photo, Phone, StatusId etc.

Reply:
return ["value", "Email", "Skype", "Job", "Company", "Address"].some((key) => contact[key].indexOf(value) !== -1) || webix.i18n.longDateFormatStr(contact.Birthday).toLowerCase().indexOf(value) !== -1;